celery/backends/mongodb.py

```diff
@@ -181,6 +181,8 @@ def _store_result(self, task_id, result, state,
                 self.current_task_children(request),
             ),
         }
+        if request and getattr(request, 'parent_id', None):
```

Comment:
Are there any other backends where this is required?

Reply:
This problem influences all kinds of backends, not only MongoDB backend.

Reply:
If that's the case, we might need to patch all the back end like this way? @tothegump

Reply:
yes, if the backend have its own `_store_result` method, or it will use data in `celery/backends/base.py`.
Three files in backends have `_store_result` method: 
`backends/mongodb.py`, `backends/database/__init__.py`, `backends/cassandra.py`

I think, I missed database and cassandra cases.

Reply:
no problem you fix one, we will try to fix the others! one back end in One PR.

Reply:
@auvipy We don't have an issue about this? right?